src/registrar/templates/domain_nameservers.html

```diff
@@ -0,0 +1,52 @@
+{% extends "domain_base.html" %}
+{% load static field_helpers%}
+
+{% block title %}Domain name servers | {{ domain.name }} | {% endblock %}
+
+{% block domain_content %}
+  {# this is right after the messages block in the parent template #}
+  {% for form in formset %}
+    {% include "includes/form_errors.html" with form=form %}
+  {% endfor %}
+
+  <h1>Domain name servers</h1>
+
+  <p>Before your domain can be used we'll need information about your domain
+  name servers.</p>
+
+  <p><a class="usa-link" href="{% url "todo" %}">Get help with domain servers.</a></p>
+
+  {% include "includes/required_fields.html" %}
+
+  <form class="usa-form usa-form--large" method="post" novalidate id="form-container">
+    {% csrf_token %}
+    {{ formset.management_form }}
+
+    {% for form in formset %}
+    <div class="server-form">
+    {% with sublabel_text="Example: ns"|concat:forloop.counter|concat:".example.com" %}
+      {% if forloop.counter <= 2 %}
+        {% with attr_required=True %}
+          {% input_with_errors form.server %}
+        {% endwith %}
+      {% else %}
+        {% input_with_errors form.server %}
+      {% endif %}
+    {% endwith %}
+    </div>
+    {% endfor %}
+
+    <button type="submit" name="submit_button" class="usa-button usa-button--unstyled display-block" id="add-form">
```

Comment:
@neilmb, myself, @brandonlenz and @rachidatecs  were wondering why the type here is submit? Couldn't it be type='button'?

Reply:
You're right. The `type="submit"` was still present from a previous approach (that we use in e.g.  `registrar/templates/application_dotgov_domain.html`). I'll change it to `button`.